sklearn/covariance/_elliptic_envelope.py

```diff
@@ -7,6 +7,8 @@
 from ..utils.validation import check_is_fitted
 from ..metrics import accuracy_score
 from ..base import OutlierMixin
+from ..utils._param_validation import Interval
+from numbers import Real
```

Comment:
Move this import on the top since this a standard Python lib import.

Reply:
done